src/sentry/testutils/pytest/kafka.py

```diff
@@ -115,6 +126,62 @@ def scope_consumers():
                 _log.warning("Failed to cleanup consumer %s", consumer_name)
 
 
+def get_config(
```

Comment:
This is copypasted from the main codebase into this pytest utility since that's the only place it's used now